src/_topbars.scss

```diff
@@ -79,21 +79,44 @@
 /* Topbar: List
 /* ========================================================================== */
 
-.topbar-list-container {
+.topbar-toggle {
+  background-color: transparent;
+  border: 0;
+
+  @media (min-width: $screen-md-min) {
+    display: none;
+  }
+}
+
+.topbar-menu {
   margin-left: auto;
 }
 
 .topbar-list {
-  flex-basis: calc(100% + 24px * 2);
   list-style-type: none;
-  margin: 12px -24px 0;
   padding: 0;
 
+  @media (max-width: $screen-md-min) {
+    background-color: $primary;
+    color: $white;
+    overflow: hidden;
+    left: 0;
+    margin: 0;
+    max-height: 0;
+    position: absolute;
+    right: 0;
+    top: 60px;
+    z-index: 2;
+  }
+
   @media (min-width: $screen-md-min) {
     align-items: center;
     display: flex;
-    flex-basis: auto;
-    margin: 0 0 0 auto;
+  }
+
+  &[aria-expanded="true"] {
```

Comment:
Can we have a CSS class here instead of an attribute? That doesn't mean we will not have aria on our markup, it only means that our CSS doesn't depend on the HTML as much.